internal/tools/securitypolicy/main.go

```diff
@@ -13,9 +13,11 @@ import (
 )
 
 var (
-	configFile = flag.String("c", "", "config path")
-	outputType = flag.String("t", "", "[rego|json]")
-	outputRaw  = flag.Bool("r", false, "whether to print the raw output")
+	configFile        = flag.String("c", "", "config path")
+	outputType        = flag.String("t", "", "[rego|json|fragment]")
+	fragmentNamespace = flag.String("n", "", "fragment namespace")
```

Comment:
Just wondering if there is a reason why the fragment Namespace isn't included in the FragmentConfig struct? Also, SVN is included in FragmentConfig and as an input, just wondering the purpose of this?

Also, you should update the example in internal/tools/securitypolicy/README to show what you expect the config file to look like with these changes.

Reply:
I'm happy to provide samples if we want them. I think we've been hesitating to make the change as per @anmaxvl until the interface is locked in, with all the enforcement points etc. However, if we want to do that incrementally I'm happy to do it, please let me know what you prefer.

Reply:
I've updated the README as requested. Please let me know if this is what you had in mind, and if so I'll resolve the conversation.

Reply:
Yes, this is exactly what I had in mind. Thanks!